ynr/apps/parties/api/next/serializers.py

```diff
@@ -51,6 +51,7 @@ class Meta:
             "emblems",
             "descriptions",
             "legacy_slug",
+            "created",
```

Comment:
I think we should also add the `last_updated` field to the `PartySerializer`, and then add filtering by this field too. As this is another import process that we can then change to only import recently updated objects